src/OnlySpans.PolyLeads.Api/Startup/Startup.Databases.cs

```diff
@@ -85,4 +87,66 @@ private static async Task MigrateDatabaseAsync(this WebApplication app)
 
         await dbContext.Database.MigrateAsync();
     }
+    
+    private static WebApplication SeedUserRoles(this WebApplication app)
+    {
+        using var scope = app.Services.CreateScope();
+        var provider = scope.ServiceProvider;
+
+        var roleManager = provider.GetRequiredService<RoleManager<Entities.ApplicationUserRole>>();
+        
+        var roles = new List<Entities.ApplicationUserRole>
+        {
+            new() { Name = "Admin" },
+            new() { Name = "Trade_union" },
+            new() { Name = "Headman" },
+            new() { Name = "Student" }
+        };
+        
+        foreach (var role in roles)
+        {
+            if (!roleManager.RoleExistsAsync(role.Name!).Result)
+            {
+                var result = roleManager.CreateAsync(role).Result;
+                
+                if (!result.Succeeded)
+                    throw new Exception($"Ошибка при создании роли {role.Name}.");
+            }
+        }
+
+        return app;
+    }
+
+    private static async Task SeedMainAdmin(this WebApplication app)
+    {
+        using var scope = app.Services.CreateScope();
+        var provider = scope.ServiceProvider;
+
+        var userManager = provider.GetRequiredService<UserManager<Entities.ApplicationUser>>();
+
+        var oldBerkas = await userManager.FindByNameAsync("Berkas");
+
+        if (oldBerkas != null)
+        {
+            var hasRightPassword = await userManager.CheckPasswordAsync(oldBerkas, "svo12345");
+
+            var hasAdminRole = await userManager.IsInRoleAsync(oldBerkas,"Admin");
+
+            if (hasRightPassword && hasAdminRole)
+                return;
+
+            await userManager.DeleteAsync(oldBerkas);
+        }
+
+        var mainAdmin = new Entities.ApplicationUser
```

Comment:
```suggestion
       masterUser = new Entities.ApplicationUser
```

просто переиспользуешь переменную